### DISCUSSION TITLE
Migration to dotnet-sdk 5

### DISCUSSION THREAD

**Discussion by Spixmaster** [btcpayserver/btcpayserver#2371] [General]
The Arch repositories just updated all the dotnet packages from version major 3 to 5. After the update, I wanted to start nbxplorer and btcpayserver but I got the following error message:
```
It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '3.1.0' was not found.
  - The following frameworks were found:
      5.0.3 at [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.AspNetCore.App&framework_version=3.1.0&arch=x64&rid=arch-x64
```

When will this project be updated to the recent SDK? I am not familiar with C# development at all so I have no clue whether it would take much work.

**Comment by bolatovumar** [comment]
@Spixmaster why not install the `3.1.0` version yourself? As far as I know there are no plans to migrate to version 5 in the immediate future. It will probably happen at some point but you may have to wait for a while.

**Reply by Spixmaster** [reply]
Yes, I did that. Nevertheless, it is good to know that version major 3 will be used for a longer time.

**Reply by bolatovumar** [reply]
ping @NicolasDorier who might have more insight on this

**Comment by Kukks** [comment]
We'll wait for an LTS version of net5 first
